Documentation/data-collection.md

```diff
@@ -594,6 +594,21 @@ data:
     #
     # The current amount of CPU used by Application Services products, aggregated by product name.
     - '{__name__="appsvcs:cores_by_product:sum"}'
+    #
+    # owners: (@openshift/team-build-api)
+    #
+    # openshift:openshift_csi_share_configmap_total:sum measures total amount of config maps shared by csi shared resource driver.
```

Comment:
this doesn't match the metric name

Reply:
i think i forgot to rebuild the docs. thanks!